src/EnergyPlus/ZoneEquipmentManager.hh

```diff
@@ -129,7 +129,7 @@ namespace ZoneEquipmentManager {
                                     Optional_int_const EquipPriorityNum = _ // index in PrioritySimOrder for this update
     );
 
-    void CalcZoneMassBalance();
+    void CalcZoneMassBalance(bool const FirstHVACIteration);
```

Comment:
Where did you use this new bool?

Reply:
Here in the [zone balance check](https://github.com/NREL/EnergyPlus/pull/7059/files#diff-e954944c50c5b52c35f6c9f2d1ad3bf5R4845).